### ISSUE TITLE
Orange Image Analytics - fails to load new set of images

### ISSUE THREAD

**Issue by DataGeo** [biolab/orange3#3916]
##### Orange version
Orange version 3.22.0

##### Expected behavior
Load Images

##### Actual behavior
Raises an error - images not loaded

##### Steps to reproduce the behavior

Start new project,
drag Import images widget, 
change file path to new file path for new set of images

##### Additional info (worksheets, data, screenshots, ...)
Exception:
OSError: [WinError 6] The handle is invalid: True
Module:
genericpath:96
Widget Name:
Import Images
Widget Module:
orangecontrib.imageanalytics.widgets.owimageimport:283
Widget Scheme:
C:\Users\XXX~1\AppData\Local\Temp\ows-hsexfx2w.ows.xml
Version:
3.22.0
Environment:
Python 3.6.8 on Windows 10 10.0.17134 AMD64
Installed Packages:
AnyQt==0.0.10, Bottleneck==1.2.1, CacheControl==0.12.5, Orange3-ImageAnalytics==0.3.0, Orange3==3.22.0, Pillow==5.4.1, PySocks==1.7.0, XlsxWriter==1.1.8, asn1crypto==0.24.0, brotlipy==0.7.0, certifi==2019.6.16, cffi==1.12.3, chardet==3.0.4, cryptography==2.7, cycler==0.10.0, decorator==4.4.0, docutils==0.14, entrypoints==0.3, h2==2.6.2, hpack==3.0.0, hyperframe==3.2.0, hypertemp==0.8.0, idna==2.8, joblib==0.13.2, keyring==18.0.0, keyrings.alt==3.1.1, kiwisolver==1.1.0, lap==0.4.0, llvmlite==0.29.0, lockfile==0.12.2, matplotlib==3.1.0, mkl-fft==1.0.12, mkl-random==1.0.2, mkl-service==2.0.2, msgpack==0.6.1, ndf==0.1.4, networkx==2.3, numba==0.44.1, numpy==1.16.4, olefile==0.46, openTSNE==0.3.8, pandas==0.24.2, pip==19.1.1, pyOpenSSL==19.0.0, pycparser==2.19, pynndescent==0.3.0, pyparsing==2.4.0, pyqtgraph==0.10.0, python-dateutil==2.8.0, python-louvain==0.13, pytz==2019.1, pywin32==223, requests==2.22.0, rfc3986==1.3.2, scikit-learn==0.21.2, scipy==1.2.1, serverfiles==0.3.0, setuptools==41.0.1, six==1.12.0, tornado==6.0.2, urllib3==1.24.2, wheel==0.33.4, win-inet-pton==1.1.0, wincertstore==0.2, xlrd==1.2.0
Machine ID:
XXXX
Stack Trace:
Traceback (most recent call last):
  File "C:\Users\XXX\AppData\Local\Orange\lib\site-packages\orangecontrib\imageanalytics\widgets\owimageimport.py", line 283, in __runOpenDialog
    self.setCurrentPath(dirpath)
  File "C:\Users\XXX\AppData\Local\Orange\lib\site-packages\orangecontrib\imageanalytics\widgets\owimageimport.py", line 343, in setCurrentPath
    and os.path.samefile(os.path.isdir(self.recent_paths[0].abspath), path):
  File "C:\Users\XXX\AppData\Local\Orange\lib\genericpath.py", line 96, in samefile
    s1 = os.stat(f1)
OSError: [WinError 6] The handle is invalid: True

**Comment by DataGeo** [comment]
My only current work around for this problem is to uninstall and then reinstall Orange!!!
I have installed Orange as a standalone using the Miconda installation. Im running Python separately using Enthought Canopy Version: 2.1.9.3717. This is a work around from a previous issue with Orange. Previously I was running Python with the Anaconda installation and Orange as a package under Anaconda. This worked fine till I tried to install the Image analytics Add-on which refused to load.
I would really appreciate any suggestions how to get around my current issue. 
Many thanks.
